tests/api/v1/datadog/features/monitor.feature

```diff
@@ -0,0 +1,49 @@
+@v1 @endpoint(monitors) @todo
+Feature: Monitors
+  [Monitors](https://docs.datadoghq.com/monitors) allow you to watch a
+  metric or check that you care about, notifying your team when some defined
+  threshold is exceeded. Refer to the Creating Monitors page for more
+  information on creating monitors.
+
+  Background:
+    Given a valid "apiKeyAuth" key in the system
+    And a valid "appKeyAuth" key in the system
+    And an instance of "Monitors" API
+
+  Scenario: Get all monitor details returns "OK" response
+    Given new "ListMonitors" request
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: Create a monitor returns "OK" response
+    Given new "CreateMonitor" request
+    And body {}
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: Check if a monitor can be deleted returns "OK" response
+    Given new "CheckCanDeleteMonitor" request
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: Validate a monitor returns "OK" response
+    Given new "ValidateMonitor" request
+    And body {}
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: Delete a monitor returns "OK" response
+    Given new "DeleteMonitor" request
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: Get a monitor's details returns "OK" response
+    Given new "GetMonitor" request
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: Edit a monitor returns "OK" response
+    Given new "UpdateMonitor" request
+    And body {}
+    When the request is sent
+    Then the status is 200 OK
```

Comment:
Should this be 400? (Or should I just be looking for semantics currently?)

Reply:
It's marked with `@todo` tag. We need to fill-up the content.